doc/api/assert.md

```diff
@@ -348,7 +348,7 @@ const callsfunc = tracker.calls(func);
 callsfunc(1, 2, 3);
 
 assert.deepStrictEqual(tracker.getCalls(callsfunc),
-                       [{ thisArg: this, arguments: [1, 2, 3 ] }]);
+                       [{ thisArg: undefined, arguments: [1, 2, 3 ] }]);
```

Comment:
Note that in ESM, `this === undefined`, so the code was technically correct as is. being explicit doesn't hurt though.
Should we fix the odd spacing while we're changing this line?
```suggestion
                       [{ thisArg: undefined, arguments: [1, 2, 3] }]);
```

Reply:
Thank you for kind explanation. Use `this` for ESM and use `undefined` for CJS.
Plus, fix odd spacing in `arguments` for both ESM and CJS.

Reply:
@aduh95 Applied your suggestion. Could you please review again?